app/controllers/users/phones_controller.rb

```diff
@@ -11,6 +11,8 @@ class PhonesController < ApplicationController
     before_action :allow_csp_recaptcha_src, if: :recaptcha_enabled?
     before_action :confirm_recently_authenticated
 
+    helper_method :in_multi_mfa_selection_flow?
```

Comment:
What's this for?

Reply:
for the back or cancel option it needs this method. Goign to find a better way to do this though. i dont like its needed in the controller that happens.